Common/src/gRPC/Services/GrpcVersionsService.cs

```diff
@@ -28,15 +28,25 @@
 
 namespace ArmoniK.Core.Common.gRPC.Services;
 
+/// <summary>
+///   Service to provide version information for the API and Core components.
+/// </summary>
```

Comment:
Would that work?
```suggestion
/// <inheritdoc cref="Versions.VersionsBase" />
```

Reply:
but `/// <inheritdoc cref="Versions" />` gives our comment from API.